core/controllers/base_test.py

```diff
@@ -519,7 +519,8 @@ def test_every_method_has_decorator(self):
             # Following handler are present in base.py where acl_decorators
             # cannot be imported.
             if (handler.__name__ == 'LogoutPage' or
-                    handler.__name__ == 'Error404Handler'):
+                    handler.__name__ == 'Error404Handler' or
+                    handler.__name__ == 'EditorLogoutHandler'):
```

Comment:
Why add it here? It doesn't have the problem that's noted in the comment above...

Reply:
yes, i will create a new if statement.